classes/DefaultArgumentRemoverReturnTypes100.php

```diff
@@ -38,7 +38,7 @@ public function matches(Invocation $invocation) : bool
                 $iClass ? Invocation::class : Invocation\StaticInvocation::class
             );
         } else {
-            MockFunctionGenerator::removeDefaultArguments($invocation->parameters);
+            MockFunctionGenerator::removeDefaultArguments($invocation->parameters());
```

Comment:
fix only vars can be passed as reference.

```suggestion
            $paramaters = $invocation->parameters();
            MockFunctionGenerator::removeDefaultArguments($paramaters);
```